Sources/NetworkProtection/NetworkProtectionDeviceManager.swift

```diff
@@ -65,10 +65,8 @@ public protocol NetworkProtectionDeviceManagement {
     typealias GenerateTunnelConfigurationResult = (tunnelConfiguration: TunnelConfiguration, server: NetworkProtectionServer)
 
     func generateTunnelConfiguration(resolvedSelectionMethod: NetworkProtectionServerSelectionMethod,
-                                     includedRoutes: [IPAddressRange],
-                                     excludedRoutes: [IPAddressRange],
```

Comment:
Included and excluded routes will no longer by passed by the client or by settings.  They'll be calculated by the extension based on settings.

If we ever need to add support for custom IP ranges, we'll just add the appropriate overrides in settings but still calculate all the routing here.